### ISSUE TITLE
"from" keyword doesn't seem to be taken into account properly

### ISSUE THREAD

**Issue by romainreuillon** [alexarchambault/coursier#340]
The from sbt keyword as for instance in:
"ccl.northwestern.edu" % "netlogo" % netLogo5Version % "provided" from s"https://github.com/NetLogo/NetLogo/releases/download/$netLogo5Version/NetLogo.jar"

should be top priority over the local and remote resolutions. It was working fine with M13 but with M14 coursier try to reslove the dependency even if the location is specified using "from".

**Comment by alexarchambault** [comment]
It was sometimes done this way by accident, in M13, the correct behavior seems to give the URL a low priority, per the SBT scaladoc of `ModuleID.from`:
```
  /**
   * Specifies a URL from which the main artifact for this dependency can be downloaded.
   * This value is only consulted if the module is not found in a repository.
   * It is not included in published metadata.
   */
```

**Comment by romainreuillon** [comment]
It's weird, it doesn't work the way it is described here in SBT.When I disable coursier, the resolution append as I would expected (from is given highest priority and the compilation is successful, whereas it doesn't work using coursier M14).

**Comment by romainreuillon** [comment]
Actually I think I misinterpreted the SBT error. The "from" keyword doesn't seem to be taken into account at all by coursier. In my dependencies I have:
`libraryDependencies += "ccl.northwestern.edu" % "netlogo" % netLogo5Version % "provided" from s"https://github.com/NetLogo/NetLogo/releases/download/$netLogo5Version/NetLogo.jar"`

and I get this error:
`coursier.ResolutionException: Encountered 1 error(s) in dependency resolution:
  ccl.northwestern.edu:netlogo:5.3.1:
    not found:
      /home/reuillon/.ivy2/local/ccl.northwestern.edu/netlogo/5.3.1/ivys/ivy.xml
      https://repo1.maven.org/maven2/ccl/northwestern/edu/netlogo/5.3.1/netlogo-5.3.1.pom
      https://oss.sonatype.org/content/repositories/snapshots/ccl/northwestern/edu/netlogo/5.3.1/netlogo-5.3.1.pom
      https://oss.sonatype.org/content/repositories/releases/ccl/northwestern/edu/netlogo/5.3.1/netlogo-5.3.1.pom
      https://github.com/NetLogo/NetLogo/releases/download/5.3.1/
[error] (ccl-northwestern-edu-netlogo5/*:coursierResolution) coursier.ResolutionException: Encountered 1 error(s) in dependency resolution:
[error]   ccl.northwestern.edu:netlogo:5.3.1:
[error]     not found:
[error]       /home/reuillon/.ivy2/local/ccl.northwestern.edu/netlogo/5.3.1/ivys/ivy.xml
[error]       https://repo1.maven.org/maven2/ccl/northwestern/edu/netlogo/5.3.1/netlogo-5.3.1.pom
[error]       https://oss.sonatype.org/content/repositories/snapshots/ccl/northwestern/edu/netlogo/5.3.1/netlogo-5.3.1.pom
[error]       https://oss.sonatype.org/content/repositories/releases/ccl/northwestern/edu/netlogo/5.3.1/netlogo-5.3.1.pom
[error]       https://github.com/NetLogo/NetLogo/releases/download/5.3.1/
[error] Total time: 143 s, completed 20 sept. 2016 14:46:20`

And it works when coursier is disabled.

**Comment by alexarchambault** [comment]
Should be fixed by https://github.com/alexarchambault/coursier/issues/373, included in `1.0.0-M14-7` and `1.0.0-M14-8` in particular.
